breadcord/helpers.py

```diff
@@ -165,7 +166,7 @@ def decorator(func: ItemCallbackType[V, Button[V]]) -> ItemCallbackType[V, Butto
 
 
 def simple_transformer(to: type[_T]) -> Callable[[type[_Transformer]], _Transformer]:
-    """A decorator for discord.py transformers which makes them easier to use in type annotations.
+    """Decorator for discord.py transformers to make them easier to use in type annotations.
```

Comment:
See previous comment @ L27.

```suggestion
    """A decorator for discord.py transformers to make them easier to use in type annotations.
```